user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Tables/Table_component.md

```diff
@@ -33,25 +33,76 @@ In the *Layout* tab for this component, the *Column filters* option is available
 
 - You can remove a column filter again by selecting *No color* instead of a specific color.
 
-You can **resize the columns** of the table by dragging the edges of the column headers. From DataMiner 10.2.0/10.1.8 onwards, you can also change the order of the columns by dragging the column headers to a different position.
+From DataMiner 10.4.1/10.5.0 onwards<!-- RN 37522 -->, in the *Layout* tab, the *Column appearance* option is available, which allows you to customize the appearance of a column.
+
+- *Column appearance > Select a column*: Allows you to alter the appearance of a column using presets. You can configure this option as follows:
+
+  1. In the dropdown list, select the column you want to change.
+
+     A preview of the column's appearance is displayed underneath the name of the column.
+
+  1. Click the preview to expand a list of available presets. These show what the column would look like if that preset was selected.
+
+  1. Select one of the presets to apply it to the column.
+
+     Available presets:
+
+     - Left: Align the content to the left.
+
+     - Center: Align the content in the center.
+
+     - Right: Align the content to the right.
+
+     - Icon: Display an icon instead of text.
+
+     - Link: Make the text appear as a link.
```

Comment:
I'm currently working on a DMA running DataMiner version 10.4.1, but when I look at these presets, I don't have the option to make the text appear as a link. 

![image](https://github.com/SkylineCommunications/dataminer-docs/assets/113898085/5f57ef99-1944-472e-8a99-833fd20da0a5)

Any idea why this might be, @GillesBara? Is this setting not available yet in 10.4.1?

Reply:
Update: Hyperlink option is only available in low-code apps.